### DISCUSSION TITLE
Eslint plugin for Biome formatter

### DISCUSSION THREAD

**Discussion by yann-combarnous** [biomejs/biome#1096] [Ideas]
For progressive adoption, it would be nice to be able to use Biome as formatter for eslint, similar to how eslint-plugin-prettier integrates prettier there.

**Comment by ematipico** [comment]
Hey @yann-combarnous do you want to try to build something? We could include it in our organization if the library doesn't go in conflict with Biome's linter

**Reply by kbrilla** [reply]
hi, how can you execute biome formatter from js script in node? let's say I want to format arbitary `a.js` file?

**Reply by ematipico** [reply]
https://www.npmjs.com/package/@biomejs/js-api

**Comment by arianrhodsandlot** [comment]
I just made one called eslint-plugin-biome-x https://github.com/arianrhodsandlot/eslint-plugin-biome-x if you are still seeking for that. It can not only format the code but can also report the issues raised by biome lint.

**Reply by kbrilla** [reply]
Great, was actually working on homebrew solution but will love to have a look at your take on it! Thank You
